test_label_lines/test_label_lines.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+import os
```

Comment:
These personal tests should not be part of the commit.  Please write pytest tests that don't need data input.

Reply:
Yes, we'll delete the temporary commit that adds them once we add the pytests.